cake-recipe.txt

```diff
@@ -8,6 +8,7 @@ Ingredients:
  - 2 eggs
  - 1 cup milk
  - 1/2 cup canola oil
```

Comment:
Remove the Canola oil, as you are replacing with Avacado oil; we don't want 2x the oil